package-res/reportviewer/reportviewer-prompt.js

```diff
@@ -302,6 +322,7 @@ define(['common-ui/util/util', 'pentaho/common/Messages', "dijit/registry", 'com
         // Never send the session back. This is generated by the server.
         delete options['::session'];
         // -------------
+        var options = me._buildReportContentOptions(this._getParameterDefinitionRenderMode(promptMode));
```

Comment:
@krivera-pentaho looks like forgot to remove old code above: https://github.com/krivera-pentaho/pentaho-platform-plugin-reporting/blob/master/package-res/reportviewer/reportviewer-prompt.js#L314-L323